src/status_im/multiaccounts/login/core.cljs

```diff
@@ -472,7 +472,7 @@
   "Decides which root should be initialised depending on user and app state"
   [db]
   (if (get db :tos/accepted?)
-    (re-frame/dispatch [:init-root :chat-stack])
+    (re-frame/dispatch [:init-root (if @config/new-ui-enabled? :home-stack :chat-stack)])
```

Comment:
but here we init `:home-stack` ?

Reply:
L6 - https://github.com/status-im/status-react/pull/13384/files#diff-9a40befc1fdc8639d072630daee3a96e9baf03b06ec24b05b764791d8f9ef828

Reply:
im confused, but ok i trust you :)